airflow/configuration.py

```diff
@@ -544,6 +544,26 @@ def parameterized_config(template):
 
 conf.read(AIRFLOW_CONFIG)
 
+if conf.has_option('core', 'AIRFLOW_HOME'):
```

Comment:
I am not able to understand how we get this? I understand `conf.get('core', 'airflow_home')` but do we get this from directly environment variable? If so, how is it an option in `core` section..

Shouldn't it be just
```
if 'AIRFLOW_HOME' in os.environ and conf.get('core', 'airflow_home'): 
```

Reply:
Due to https://github.com/apache/airflow/blob/d91153814872c831b07befe92c27c222b6960b02/airflow/configuration.py#L222-L224
`conf.has_option('core', 'airflow_home')` and `conf.has_option('core',  'AIRFLOW_HOME')` are the same